sktime/clustering/tests/test_time_series_k_means.py

```diff
@@ -0,0 +1,54 @@
+# -*- coding: utf-8 -*-
+import numpy as np
+
+from sktime.clustering._time_series_k_means import TimeSeriesKMeans
+
+# import os
+# import sktime
+# from sktime.clustering.evaluation._plot_clustering import plot_cluster_algorithm
+# from sktime.utils.data_io import load_from_tsfile_to_dataframe
+
+# DATA_PATH = os.path.join(os.path.dirname(sktime.__file__), "datasets/data")
+# X, Y = load_from_tsfile_to_dataframe(
+#     os.path.join(DATA_PATH, "ArrowHead/ArrowHead_TRAIN.ts")
+# )
+#
+# X_test, Y_test = load_from_tsfile_to_dataframe(
+#     os.path.join(DATA_PATH, "ArrowHead/ArrowHead_TEST.ts")
+# )
+
+
+def __run_test(model: TimeSeriesKMeans):
+    n, sz = 100, 10
+    rng = np.random.RandomState(0)
+    X = rng.randn(n, sz)
+    rng = np.random.RandomState(1)
+    X_test = rng.randn(n, sz)
+
+    model.fit(X)
```

Comment:
I suggest to automate these tests, using `all_estimators` and a common set of unit tests for API compliance, especially types/dimension of input and output to the main interface points (`fit`, `predict`)